dict/portuguese.yml

```diff
@@ -2,290 +2,290 @@
-    'SD': [Sudão, República do Sudão, Sudanês*, Cartum]
-    'SS': [Sudão do SOUTH, Sudão Meridional, República do Sudão do SOUTH, SOUTH-Sudanês*, Juba]
+    'MA': [Marrocos, Marroquin*, Rabat, Rabate]
+    'SD': [Sudão, Sudanês*, Cartum]
+    'SS': [Sudão do Sul, Sudão Meridional, Sul-Sudanês*, Juba]
     'TN': [Tunísia, República Tunisina, Tunisian*, Tunisin*, Tunes, Túnis]
 
   SOUTH:
-    'BW': [Botswana, Botsuana, República do Botswana, República do Botsuana, Bechuan*, Botsuanense*, Botsuan*, Botsuanês*, Gaborone]
-    'LS': [Lesoto, Lessoto, Reino do Lesoto, Lesotian*, Lestotense*, Lesoto*, Maseru]
-    'NA': [Namíbia, República da Namíbia, Namibian*, Vinduque]
-    'SZ': [Essuatíni, Reino de Essuatíni, Essuatinian*, Lobamba, Mebabane, Mbabane]
-    'ZA': [África do SOUTH, República da África do SOUTH, SOUTH-African*, Austro-African*, Cidade do Cabo, Joanesburgo, Johannesburgo, Pretória]
+    'BW': [Botswana, Botsuana, Bechuan*, Botsuanense*, Botsuan*, Botsuanês*, Gaborone]
+    'LS': [Lesoto, Lessoto, Lesotian*, Lestotense*, Lesoto*, Maseru]
+    'NA': [Namíbia, Namibian*, Vinduque]
+    'SZ': [Essuatíni, Essuatinian*, Lobamba, Mebabane, Mbabane]
+    'ZA': [África do Sul, Sul-African*, Austro-African*, Cidade do Cabo, Joanesburgo, Johannesburgo, Pretória]
 
   WEST:
     'BF': [Burquina Fasso, Burquina Faso, Burquina, Burquinense*, Burquinabê, Burquinês*, Burquin*, Uagadugu]
-    'BJ': [Benim, República do Benim, Beninês*, Beninense*, Porto Novo]
+    'BJ': [Benim, Beninês*, Beninense*, Porto Novo]
     'CI': [Costa do Marfim, República de Côte d'Ivoire, Costa-Marfinense*, Marfinense*, Ebúrne*, Ivoiriense*, Iamussucro, Abidjã, Abijã, Abijão, Abidjan]
-    'CV': [Cabo Verde, República de Cabo Verde, Cabo-Verdian*, Praia]
-    'GH': [Gana, República do Gana, Ganês*, Ganense*, Acra]
-    'GM': [Gâmbia, República da Gâmbia, Gambian*, Banjul]
-    'GN': [Guiné, República da Guiné, Guinean*, Guineense*, Conacri]
-    'GW': [Guiné-Bissau, República da Guiné-Bissau, Guineense*, Bissau, Sector Autónomo de Bissau]
-    'LR': [Libéria, República da Libéria, Liberian*, Monróvia]
+    'CV': [Cabo Verde, Cabo-Verdian*, Praia]
+    'GH': [Gana, Ganês*, Ganense*, Acra]
+    'GM': [Gâmbia, Gambian*, Banjul]
+    'GN': [Guiné, Guinean*, Guineense*, Conacri]
+    'GW': [Guiné-Bissau, Guineense*, Bissau, Sector Autónomo de Bissau]
+    'LR': [Libéria, Liberian*, Monróvia]
     'ML': [Mali, Máli, Malian*, Malinês*, Malês*, Bamaco]
-    'MR': [Mauritânia, República Islâmica da Mauritânia, Mauritan*, Nuaquexote, Nuaquechote]
-    'NE': [Níger, República do Níger, Nigerin*, Nigerenseé*, Niamei, Niamey]
-    'NG': [Nigéria, República Federal da Nigéria, Nigerian*, Abuja, Lagos]
+    'MR': [Mauritânia, Mauritan*, Nuaquexote, Nuaquechote]
+    'NE': [Níger, Nigerin*, Nigerenseé*, Niamei, Niamey]
+    'NG': [Nigéria, Nigerian*, Abuja, Lagos]
     'SH': [Santa Helena, Santo, Santa-Helenense*, Jamestown, Cidade de Jamestown]
```

Comment:
Do we need "Cidade de Jamestown"?

Reply:
Same thing as the previous case. I'll delete this redounded occurrences.